accounts/migrations/0001_initial.py

```diff
@@ -8,17 +8,45 @@
 class Migration(SchemaMigration):
 
     def forwards(self, orm):
+        # Adding model 'TwitterInfo'
```

Comment:
which I probably shouldn't have done.

On Fri, Jan 22, 2016 at 3:30 PM, Kelly Shen <kellyshen.cs@gmail.com> wrote:

> oh yes, when I redid migration yesterday, I reverted everything back to
> original, made the same changes in models, views, and other files, and did
> migration - the 0001_initial file didn't get changed, but since in my older
> version, I had all the twitter info, delicious info, etc. in 0001_initial,
> I manually changed the file.
>
> On Fri, Jan 22, 2016 at 3:14 PM, Amy Zhang <notifications@github.com>
> wrote:
>
>> In accounts/migrations/0001_initial.py
>> <https://github.com/haystack/eyebrowse-server/pull/141#discussion_r50585274>
>> :
>>
>> > @@ -8,17 +8,45 @@
>> >  class Migration(SchemaMigration):
>> >
>> >      def forwards(self, orm):
>> > +        # Adding model 'TwitterInfo'
>>
>> if someone checks out a new copy, the current models.py are up-to-date so
>> no need to do any migrations. You're right about someone being behind in
>> the code history though. In any event, we should not be editing older
>> migrations - just add the changes to the models in the new migration file
>> (0008_).
>> … <#-1567165571_1526733385_>
>> On Fri, Jan 22, 2016 at 3:08 PM, David Karger <notifications@github.com>
>> wrote: In accounts/migrations/0001_initial.py <#141 (comment)
>> <https://github.com/haystack/eyebrowse-server/pull/141#discussion_r50584428>>
>> : > @@ -8,17 +8,45 @@ > class Migration(SchemaMigration): > > def
>> forwards(self, orm): > + # Adding model 'TwitterInfo' I don't think
>> migrations are "just a record". I think if someone is behind in the code
>> history and does a pull, the new migrations get run to bring their database
>> in line with the current. Similarly if someone checks out a whole new copy,
>> I think the migration files are the way the db gets set up? …
>> <#1749322771_> On 01/22/2016 02:37 PM, Amy Zhang wrote: In
>> accounts/migrations/0001_initial.py <#141
>> <https://github.com/haystack/eyebrowse-server/pull/141> (comment) <#141
>> (comment)
>> <https://github.com/haystack/eyebrowse-server/pull/141#discussion_r50580575>>>:
>> > @@ -8,17 +8,45 @@ > class Migration(SchemaMigration): > > def
>> forwards(self, orm): > + # Adding model 'TwitterInfo' Can you just revert
>> all your changes to 0001_initial.py for the sake of this pull request? I
>> don't see why we need these changes. After all migration files are just to
>> keep a record of how the db has been migrated over a time. No one should be
>> running these files going forward. — Reply to this email directly or view
>> it on GitHub <
>> https://github.com/haystack/eyebrowse-server/pull/141/files#r50580575>.
>> — Reply to this email directly or view it on GitHub <
>> https://github.com/haystack/eyebrowse-server/pull/141/files#r50584428>.
>> -- Amy X. Zhang | Ph.D. student at MIT CSAIL |
>> http://people.csail.mit.edu/axz | @amyxzh
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/haystack/eyebrowse-server/pull/141/files#r50585274>.
>>
>
>